src/inc/quic_datapath.h

```diff
@@ -152,6 +152,7 @@ typedef struct CXPLAT_ROUTE {
     uint8_t LocalLinkLayerAddress[6];
     uint8_t NextHopLinkLayerAddress[6];
     BOOLEAN Resolved; // When TRUE, LocalLinkLayerAddress and NextHopLinkLayerAddress are valid.
+    uint8_t Queue;
```

Comment:
Why does this need to be here?

Reply:
My main issue with this is that this struct is currently cleanly defined as "Structure to represent a network route", but this new field has nothing [modulo some philosophical arguments about the queue being part of the effective "route" of a packet] to do with that. Is there some more generic name for this struct that would be appropriate? Or, could we just plumb the queue ID as a parameter in the appropriate functions?

Also, based on names chosen elsewhere in this change, this name should be "QueueId" rather than "Queue".

Reply:
I've updated the name. This is necessary so that we match RSS on the TX side.